cspell-dict.txt

```diff
@@ -0,0 +1,261 @@
+abolve
```

Comment:
Can we put these files somewhere other than at the root level?

Reply:
Yeah, moved config and dictionary to a `.cspell` dir. Also got the vscode extension pointed at the config 👍